frontend/catalyst/compiled_functions.py

```diff
@@ -34,7 +34,7 @@
     get_decomposed_signature,
     typecheck_signatures,
 )
-from catalyst.utils import wrapper  # pylint: disable=no-name-in-module
+from catalyst.utils import wrapper
```

Comment:
This also need to be here because the wrapper module doesn't exist until you build catalyst.